hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -0,0 +1,639 @@
+
+        { bits: "10:8",
+          name: "OPERATION",
+          desc: "Key manager operation selection. All values not enumerated below behave the same as disable",
+          resval: "1"
+          enum: [
+            { value: "0",
+              name: "Advance",
+              desc: '''
+                Advance key manager state.
+
+                Advances key manager to the next stage.
+                If key manager is already at last functional state, the advance operation moves to
+                behaves similar to the disable operation.
+                '''
+            },
+            { value: "1",
+              name: "Generate ID",
+              desc: '''
+                Generates an identity seed from the current state.
+                '''
+            },
+            { value: "2",
+              name: "Generate SW Output",
+              desc: '''
+                Generates a key manager output that is visible to software from the current state.
+                '''
+            },
+            { value: "3",
+              name: "Generate HW Output",
+              desc: '''
+                Generates a key manager output that is visible only to hardware crypto blocks.
+                '''
+            },
+            { value: "4",
+              name: "Disable",
+              desc: '''
+                Disables key manager operation and moves it to the disabled state.
+
+                Note the disabled state is terminal and cannot be recovered without a reset.
+                '''
+            },
+          ]
+        },
+
+        { bits: "13:12",
+          name: "DEST_SEL",
+          desc: '''
+            When the OPERATION field is programmed to generate HW output, this field selects
+            the appropriate crypto cipher target.
+          ''',
+          resval: "0"
+          enum: [
+            { value: "0",
+              name: "None",
+              desc: '''
+                No target selected
+                '''
+            },
+            { value: "1",
+              name: "AES",
+              desc: '''
+                AES selected
+                '''
+            },
+            { value: "2",
+              name: "HMAC",
+              desc: '''
+                HMAC selected
+                '''
+            },
+            { value: "3",
+              name: "KMAC",
+              desc: '''
+                KMAC selected
+                '''
+            },
+          ]
+        },
+      ],
+    },
+
+    { name: "ROM_EXT_DESC_EN",
+      desc: "Register write enable for ROM_EXT_DESC",
+      swaccess: "rw0c",
+      hwaccess: "none",
+      fields: [
+        { bits: "1:0",
+          name: "EN",
+          resval: "2"
+          desc: '''
+            ROM_ext descriptor configure enable.
+            All values except for 2 are interpreted as configuration disable.
+          '''
+        },
+      ]
+    },
+    { multireg: {
+        cname: "KEYMGR",
+        name: "ROM_EXT_DESC",
```

Comment:
i might keep this for now. Since it's not clear when we will crank the key manager (in ROM or later), maybe keeping software binding and rom_desc separate is better.